tap_copper/schemas/activities_search.json

```
@@ -1 +1,112 @@
-{}
\ No newline at end of file
+{
+  "type": "object",
+  "additionalProperties": true,
+  "properties": {
+    "id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "type": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "additionalProperties": true,
+      "properties": {
+        "id": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "category": {
+          "type": [
+            "null",
+            "string"
+          ]
+        },
+        "name": {
+          "type": [
+            "null",
+            "string"
+          ]
+        }
+      }
+    },
+    "parent": {
+      "type": [
+        "null",
+        "object"
+      ],
+      "additionalProperties": true,
+      "properties": {
+        "id": {
+          "type": [
+            "null",
+            "integer"
+          ]
+        },
+        "type": {
+          "type": [
+            "null",
+            "string"
+          ]
+        }
+      }
+    },
+    "details": {
+      "type": [
+        "null",
+        "string"
+      ]
+    },
+    "user_id": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "activity_date": {
+      "type": [
+        "null",
+        "integer"
+      ]
+    },
+    "old_value": {
+      "type": [
+        "null",
+        "object",
+        "string",
+        "number",
+        "integer",
+        "boolean",
+        "array"
+      ]
+    },
+    "new_value": {
+      "type": [
+        "null",
+        "object",
+        "string",
+        "number",
+        "integer",
+        "boolean",
+        "array"
+      ]
+    },
```

This would cause column split on the destination, also as per the documentation [here](https://developer.copper.com/activities/overview.html#activity-properties) `old_value` and `new_value` should be `object` type and fallback datatype should be a `string`.
```suggestion
    "old_value": {
      "type": [
        "null",
        "object",
        "string"
      ]
    },
    "new_value": {
      "type": [
        "null",
        "object",
        "string"
      ]
    },
```

---

Suggested changes has been implemented